App/Containers/StatusCheck.tsx

```diff
@@ -40,9 +42,7 @@ class StatusCheck extends React.Component<Props, {}> {
       )
     } else {
       return (
-        <View style={{ flex: 1, alignContent: 'center', justifyContent: 'center' }}>
-          <ActivityIndicator size='large' />
-        </View>
+        <Loading color={color.brandBlue} />
```

Comment:
Display a blue spinner while we wait for the node to start. @andrewxhill see above on line 24 and 26 how the node has to be started before it will navigate to either the onboarding or main app navigation (from a cold start, that is)

Reply:
If the node isn't started, it either displays the loading spinner or an error message